src/apps.md

```diff
@@ -0,0 +1,24 @@
+---
+title: App development
```

Comment:
This title is inconsistent with the other "apps" pages. Not that everything has to be consistent... but it's something to consider. Here are the titles of all three main "apps" pages:

* **App development**
  * **Apps** in the left nav
* **Command-line and server apps**
  * Same in the left nav, except for shortening _and_ to _&_, per our custom
* **Web platform**
  * **Web apps** in the left nav

Reply:
If we switch the sidenav to Multi-platform apps as well, I think that's good for a title.

Reply:
Changed to 'Multi-platform apps'